.github/workflows/release.yml

```diff
@@ -16,11 +16,14 @@ jobs:
         with:
           node-version-file: .nvmrc
           cache: npm
-      - run: npm ci
-      - run: npm run build
-      - run: npm test
+      - uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
```

Comment:
Is this a good idea? Our CI may start failing at any point without that being associated to a commit. I prefer never using `latest` for CI stuff.

Reply:
(FYI I'm a bit of a purist on this, I never use `^` imports on `package.json` because I'd rather all changes to `node_modules` can be traced back to a specific commit in our repository)

Reply:
Used `bun-version-file: .bun-version` with `1.1.18`